tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -673,11 +676,11 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
             renderPage am.document, 9
             renderPage am.document, 10
 
-        # Now we trigger the actual action
-        anchor.scrollIntoView().then ->
+        anchor.scrollToView()
+        watchForScroll().then ->
```

Comment:
This seems wrong. If you're calling `scrollToView` before `watchForScroll` then I think `pendingTest` hasn't been initialized yet for this run. Furthermore, if you're resolving the promise instantaneously in your spy there's no reason to do this in a callback. I'm not sure what's getting tested here.